src/chart.js

```diff
@@ -0,0 +1,67 @@
+/** @jsx jsx */
+import React, { useState } from "react";
+import { jsx } from "@emotion/core";
+import Employee from "./employee";
+
+const data = [
+  {
+    id: 1,
+    name: "Christopher Roa",
+    title: "CEO",
+    parentId: null
+  },
+  {
+    id: 2,
+    name: "Frank Condezo",
+    title: "CTO",
+    parentId: 1
+  },
+  {
+    id: 3,
+    name: "Angie Gonzales",
+    title: "DEV",
+    parentId: 1
+  },
+  {
+    id: 4,
+    name: "Carlos Sandoval",
+    title: "DEV",
+    parentId: 2
+  },
+  {
+    id: 5,
+    name: "Paul Tomas",
+    title: "Designer",
+    parentId: 2
+  }
+];
+
+function Chart() {
+  const [list, setList] = useState(data);
+
+  function getChilds(id) {
+    return list.filter(employee => employee.parentId === id);
+  }
+
+  function addItem(item) {
```

Comment:
I like this one but I would rather change the name of addItem to addChild, for example.